### PR TITLE
Updates for the ece-tools 2002.1.4 release

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Added release notes
- Updated DATABASE_CONFIGURATION variable with additional example
of custom connection setting for Amazon Aurora MySQL database

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/release-notes/cloud-tools.html
- https://devdocs.magento.com/cloud/env/variables-deploy.html#database_configuration
- https://devdocs.magento.com/cloud/release-notes/cloud-tools.html

whatsnew
Published [ece-tools 2002.1.4 package release notes](https://devdocs.magento.com/cloud/release-notes/cloud-tools.html). This release fixes search engine and database validation issues that can cause deployment failure.

### CODE REVIEW

**Review by hguthrie — APPROVED** [review]

**Review by bdenham — APPROVED** [review]

File: src/cloud/env/variables-deploy.md [hunk 1]
```diff
@@ -194,6 +194,20 @@ stage:
       _merge: true
 ```
 
+The following example overwrites the default database connection with custom connection settings for an Amazon Aurora MySQL database:
```
Comment (NadiyaS) [inline src/cloud/env/variables-deploy.md:197]: I am not sure that we already should post information about Aurora. It is planned to be supported not earlier Magento 2.4.2

File: src/cloud/release-notes/ece-release-notes.md [hunk 2]
```diff
@@ -17,9 +17,26 @@ The `{{site.data.var.ct}}` package uses the following release versioning sequenc
...
+
+   ```terminal
+   Version of service `mysql` is not detected
+   ```
+
+   ```terminal
+   The current configuration is not compatible with this version of Magento.
+   Magento 2.4.1.0 does not support version "5.7" for service "mysql".
+   Service version should satisfy ">=10.2 <10.5" constraint.
+   ```
```
Comment (oshmyheliuk) [inline src/cloud/release-notes/ece-release-notes.md:35]: This shouldn't be in the current release notes as work with Aurora is not finished yet, and PR is not merged in ece-tools

Comment (oshmyheliuk) [reply src/cloud/release-notes/ece-release-notes.md:35]: In `v2002.1.4` we only fixed the issue with `SEARCH_CONFIGURATION`

Comment (meker12) [reply src/cloud/release-notes/ece-release-notes.md]: @oshmyheliuk  and @NadiyaS Removed that release note and associated doc update. Thanks for the clarification.

**Review by NadiyaS — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**